### PR TITLE
(#2963) Implement borderless full width feature cards.

### LINKED ISSUE #2963: Build repeating content blocks to custom landing page
As a user, I want to see an attractive landing page for the microsite on any device, so I can get an overview of the program and sign up

Build repeating content blocks to hold the repeating paragraph content.  These blocks will be fairly evergreen and shouldn't be updated often.

Each block should support:

1. title
2. paragraph text 
3. image 
4. orientation (location of images and text)
5. background color per comp
5. Clickable links to navigate to inner pages of site

Design assets: https://lion.box.com/s/v5q0rcfpc5ux98hv80syizzj4kt6zmdf

Comps:
Final Desktop: https://publicis.invisionapp.com/share/V3Y2YRRSGMH
Final Mobile: https://publicis.invisionapp.com/share/NKY61SDE97H
Annotated Comps: https://publicis.invisionapp.com/share/NEYL4UKXB63

### PR DESCRIPTION
**Pull request by blairlearn**
- Update paragraph type for custom landing page.
- Added three fields to paragraph type Borderless Full Width Feature card.
- Associated styling and template changes.
- Update yaml file for borderless content test.

Closes #2963 

Replacement for #3031

### CODE REVIEW

**Comment by blairlearn** [comment]
Replacement for #3031

**Comment by blairlearn** [comment]
There's a bug when `cgov_content_block` is used in a full-width card.  Rendering is falling through to `field.html.twig` and somehow injecting an extra `<h2>` element.

Props to @kate-mashkina for finding it.

**Comment by KateMashkinaNIH** [comment]
I confirm that the extra h2 tag is no longer present on the test page  (on ODE 409)

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss [hunk 1]
```diff
@@ -85,6 +132,50 @@ a.borderless-button {
...
+  }
+}
+.accent-warm {
+  background-color: $lemon;
+}
+.default {
+  background-color: $white;
+}
+.main-content .borderless-container .borderless-button-yellow,
+.main-content .borderless-container a.borderless-button-yellow,
```
Comment (arcepaul) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/src/styles/_borderless-cards.scss:158]: ewww, is this level of specificity needed?

**Review by arcepaul — APPROVED** [review]

**Pull request merged**